### PR TITLE
Department/course_group changes for compat. w/REST API changes

### PR DESCRIPTION
**Pull request by cmurtaugh**
This PR includes:
* changes to access course.department rather than course.departments, and course.course_group instead of course.course_groups (to match the way this data is returned by the REST API).
* changes the "departments" course filter to "department" (also to match a change in the API)

This PR also adds detailed logging of the requests that go through the API proxy endpoint. This will create a better record of changes that people make via CAAT tools.

This PR also removes some old  iSites text.

### CODE REVIEW

File: canvas_account_admin_tools/requirements/base.txt [hunk 1]
```diff
@@ -11,11 +11,11 @@ django-cached-authentication-middleware==0.2.2
 django-proxy==1.2.1
 django-redis-cache==3.0.1
 djangorestframework==3.13.1
-django-storages==1.12.3
-django-allow-cidr==0.4.1
+django-storages==1.13.1
+django-allow-cidr==0.5.1
```
Comment (Chris-Thornton-Harvard) [inline canvas_account_admin_tools/requirements/base.txt:15]: django-allow-cidr 0.5.0 is the latest available

File: canvas_account_admin_tools/settings/local.py [hunk 2]
```diff
@@ -125,10 +135,10 @@
...
-LOGGING['handlers']['default'] = {
-    'level': logging.DEBUG,
-    'class': 'logging.StreamHandler',
-    'formatter': 'simple',
-}
+# LOGGING['handlers']['default'] = {
+#     'level': logging.DEBUG,
+#     'class': 'logging.StreamHandler',
+#     'formatter': 'simple',
+# }
```
Comment (Chris-Thornton-Harvard) [inline canvas_account_admin_tools/settings/local.py:142]: Comment clean up

File: canvas_account_admin_tools/settings/local.py [hunk 3]
```diff
@@ -14,8 +23,9 @@
 CANVAS_EMAIL_NOTIFICATION['support_email_address'] = 'tltqaemails@g.harvard.edu'
 
 ALLOWED_HOSTS = ['*']
-INSTALLED_APPS += ['debug_toolbar', 'django_extensions']
-MIDDLEWARE += ['debug_toolbar.middleware.DebugToolbarMiddleware']
+INSTALLED_APPS += ['django_extensions']
+# INSTALLED_APPS += ['debug_toolbar']
+# MIDDLEWARE = ['debug_toolbar.middleware.DebugToolbarMiddleware'] + MIDDLEWARE
```
Comment (Chris-Thornton-Harvard) [inline canvas_account_admin_tools/settings/local.py:28]: Are we moving forward with removing debug_toolbar? If so maybe remove it from local requirements

Comment (cmurtaugh) [reply canvas_account_admin_tools/settings/local.py:28]: I'll leave it in the requirements file but add a comment here about how to enable it.

**Pull request closed without merging**